### ISSUE TITLE
Freezes after a few starts

### ISSUE THREAD

**Issue by EmEiBee** [mRemoteNG/mRemoteNG#2255]
mRemoteNG: V 1.77.1.27654 and 2-3 former versions
WIN10: 21H2 B19044.1766

With the 2-3 previous versions and this version it happens after about 10 to 20 starts (not constant) that mRemoteNG hangs when starting. The StartUp label and the question "Choose a panel" still appear, but this is empty. No main window or similar.

![mRemoteNG-freeze](https://user-images.githubusercontent.com/30090331/176109519-664c3ac1-82cd-49ca-aa44-636568754215.jpg)

If you try to end "Choose a panel" with the "X" or generate a new panel with "NEW", mRemoteNG freezes completely and can only be ended with the task manager.
A repair setup does not help. Only a complete uninstallation (with geek) and reinstallation -> import helps again for a few days.

Greetz
Micha

**Comment by Kvarkas** [comment]
Hello, seems you are using a bit old version, could you try with stable version 1.77.2? 
https://mremoteng.org/download

**Comment by EmEiBee** [comment]
... there is no 1.77.2 That's why I have use the last available preview ...

![Bild_2022-06-28_103001818](https://user-images.githubusercontent.com/30090331/176132457-a38ac0f7-53e8-4748-b691-fab4fadd442a.png)

**Comment by simonai1254** [comment]
The 1.77.3 seems to have a settings saving bug, so not sure if that's a good idea. 

The 1.77.2 works quite fine and can be downloaded here: https://github.com/mRemoteNG/mRemoteNG/releases

**Comment by EmEiBee** [comment]
Ok, thanx a lot. I will try this and let you know if the error hit me again or not ...

**Comment by Kvarkas** [comment]
1.77.3 in progress and its not a bug to save settings its disabled for a reason :D

**Comment by LuCarhub** [comment]
> 

Sorry, I don't understand: do you mean it's not enabled the possibility to save settings on last build 1.77.3.0? 
Why? Is it for this reason that at every restart of mRemoteNG it shows me to set automatic update but actually it's not saving any setting  at all (starting in fullscreen, allow only a single instance, etc.) ?
How can I save my settings?

**Comment by Kvarkas** [comment]
yes, I am rewriting part of code how savings are happening, to have better controls over options and backups in regards work about profiles implementation, a bit big peace so hard to swallow in couple of evenings but move forward, soon will make another nb with that enabled.

**Comment by LuCarhub** [comment]
Dear Kvarkas,

I appreciate your efforts, take your time. I didn't find anyone who mentioned my problem, so I thought I was alone. 
I tried to edit the xml file, but I don't know the parameters. Can you write me where I can find info to manually edit it?

**Comment by Kvarkas** [comment]
that's ok but for formal talks we have chat :) could you tell me that exactly you want to save it into it?

**Comment by LuCarhub** [comment]
As I wrote, everything I set on preferences. To be clear, I can set starting in fullscreen, allow only a single instance, check daily for new releases, etc. but when I close the program it looses my settings, and at restart is again asking me if I want to set automatic updates, and all my preferences are lost.

**Comment by Kvarkas** [comment]
cant reproduce
